src/select.tsx

```diff
@@ -106,8 +106,12 @@ const Select: React.FC<SelectProps> = ({
                 const top = selectTop + height;
                 const visibleOptions = Math.min(options.length, MAX_OPTIONS);
                 const spaceTaken = visibleOptions * 48 + PADDING_SIZE;
+                const fullOptionsSpace = options.length * 48 * PADDING_SIZE;
```

Comment:
Is this logic right? why multiplying by PADDING_SIZE?

Reply:
I'll change it and test it again

Reply:
Done